opentracing-shim/src/main/java/openconsensus/opentracingshim/SpanShim.java

```diff
@@ -125,4 +129,42 @@ public void finish(long finishMicros) {
     // TODO: Take finishMicros into account
     span.end();
   }
+
+  static String getEventNameFromFields(Map<String, ?> fields) {
+    Object eventValue = fields == null ? null : fields.get(Fields.EVENT);
+    if (eventValue != null) {
+      return eventValue.toString();
+    }
+
+    return DEFAULT_EVENT_NAME;
+  }
+
+  static Map<String, AttributeValue> convertToAttributes(Map<String, ?> fields) {
+    Map<String, AttributeValue> attrMap = new HashMap<String, AttributeValue>();
+
+    for (Map.Entry<String, ?> entry : fields.entrySet()) {
+      String key = entry.getKey();
+      Object value = entry.getValue();
+
+      // TODO - verify null values are NOT allowed.
+      if (value == null) {
```

Comment:
also ignore `Fields.EVENT`

Reply:
I was thinking about this, and I think we ought to keep it around, just in case the backend is using this `EVENT` field somehow - in the end, we are guessing the event name from it, and we are not super sure this tag (attribute now) won't be used at all.